frontend/public/components/dashboard/dashboards-page/cluster-dashboard/status-card.tsx

```diff
@@ -110,7 +115,7 @@ export const DashboardAlerts: React.FC<DashboardAlertsProps> = ({ labelSelector
   const [cv, cvLoaded] = useK8sWatchResource<ClusterVersionKind>(
     hasCVResource ? cvResource : ({} as WatchK8sResource),
   );
-
+  const canUpgrade = useCanClusterUpgrade();
```

Comment:
`clusterVersionIsEditable` below can be removed as `useCanClusterUpgrade` covers that.